README.md

```diff
@@ -20,6 +20,24 @@ Instructions in this README.md are written with an experienced Python developer
 
 - Done!
 
+### Setting up local development environment with PyEnv and VirtualEnvWrapper
+
+```
+pyenv install -v 3.9.0
+pyenv virtualenv 3.9.0 parking_permits
+pyenv local parking_permits
+pyenv virtualenvwrapper
```

Comment:
virtualenvwrapper provides a set of commands which makes working with virtual environments more pleasant. It also keeps all your virtual environments in one place.

All in all, I guess `pyenv-virtualenvwrapper` is a nice combination to use, because this way we can manage virtualenvs with pyenv and thus it gives more flexibility to work with different Python versions for example. pyenv-virtualenvwrapper is actually a pyenv plugin which provides a pyenv virtualenvwrapper command to manage your virtualenvs with virtualenvwrapper.

I guess it's kind of next level to `pyenv-virtualenv` because it will automatically activate your VirtualEnv when you go to root folder and such, so I would still use it / keep it :)